src/configure

```diff
@@ -776,14 +795,17 @@ function linux_configure_dynamic {
 
 # If configuration sets any of these variables, we will switch the external
 # math library. Here we unset them so that we can check later.
-unset MKLROOT
-unset CLAPACKROOT
-unset OPENBLASROOT
-unset MKLLIBDIR
+#TODO(kkm): Maybe allow env vars to provide defaults?
```

Comment:
i would definitely consider allowing these from the environment.

Reply:
I agree. I'm wondering what implications would be for different user's setups in the wild. Anything that was not there before is potentially breaking. There is anoter large pending rework of configure on my plate (disabling multithreaded math options, they are useless, basically), so I left this decision till then.